c++/naive_pattern_searching.cpp

```diff
@@ -0,0 +1,30 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+void patternSearch(char* pattern, char* string)
```

Comment:
- I would the like it to return the location of match not a void.
- secondly move the function definition below main and keep function declaration only above it just for clean code purposes